streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -252,93 +194,54 @@ private static void shuffle(final int[] data, final int windowSize) {
         }
     }
 
-    public static void verify(final String kafka, final Map<String, Set<Integer>> allData, final int maxRecordsPerKey) {
+    public static boolean verify(final String kafka,
+                                 final Map<String, Set<Integer>> inputs,
+                                 final int maxRecordsPerKey) {
         final Properties props = new Properties();
         props.put(ConsumerConfig.CLIENT_ID_CONFIG, "verifier");
         props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafka);
-        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class);
+        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
         props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class);
 
-        final KafkaConsumer<byte[], byte[]> consumer = new KafkaConsumer<>(props);
-        final List<TopicPartition> partitions = getAllPartitions(consumer, "echo", "max", "min", "dif", "sum", "cnt", "avg", "wcnt", "tagg");
+        final KafkaConsumer<String, byte[]> consumer = new KafkaConsumer<>(props);
+        final String[] topics = {"data", "echo", "max", "min", "dif", "sum", "cnt", "avg", "tagg"};
```

Comment:
note: "wcnt" is removed from the list. We weren't verifying it, and it turns out, we don't produce to it at all. So I guess it's a fossil from a prior refactor.